docs/fundamentals/implementations.md

```diff
@@ -10,25 +10,25 @@ A .NET app is developed for one or more *[implementations](../standard/glossary.
 
 Each implementation of .NET includes the following components:
 
-- One or more runtimes&mdash;for example, .NET Framework CLR and .NET 5 CLR.
-- A class library&mdash;for example, .NET Framework Base Class Library and .NET 5 Base Class Library.
+- One or more runtimes&mdash;for example, .NET Framework CLR and .NET 8 CLR.
+- A class library&mdash;for example, .NET Framework Base Class Library and .NET 8 Base Class Library.
 - Optionally, one or more application frameworks&mdash;for example, [ASP.NET](https://www.asp.net/), [Windows Forms](/dotnet/desktop/winforms/windows-forms-overview), and [Windows Presentation Foundation (WPF)](/dotnet/desktop/wpf/) are included in .NET Framework and .NET 5+.
 - Optionally, development tools. Some development tools are shared among multiple implementations.
 
 There are four .NET implementations that Microsoft supports:
 
-- .NET 5 (and .NET Core) and later versions
+- .NET 6 and later versions
 - .NET Framework
 - Mono
 - UWP
 
-.NET 7 is currently the primary implementation, and the one that's the focus of ongoing development. .NET 7 is built on a single code base that supports multiple platforms and many workloads, such as Windows desktop apps and cross-platform console apps, cloud services, and websites. [Some workloads](../core/whats-new/dotnet-6.md#sdk-workloads), such as .NET WebAssembly build tools, are available as optional installations.
+.NET, previously referred to as .NET Core, is currently the primary implementation. .NET (8) is built on a single code base that supports multiple platforms and many workloads, such as Windows desktop apps and cross-platform console apps, cloud services, and websites. [Some workloads](../core/whats-new/dotnet-6.md#sdk-workloads), such as .NET WebAssembly build tools, are available as optional installations.
 
-## .NET 5 and later versions
+## .NET 6 and later versions
```

Comment:
Should this one stay as ".NET 5 and later"? Or did you update it because 5 is out of support?